portfolio/src/main/webapp/index.html

```diff
@@ -13,6 +13,12 @@
         <link href="https://fonts.googleapis.com/css2?family=Merriweather:ital@1&display=swap" rel="stylesheet">
         <link href="https://fonts.googleapis.com/css2?family=Lobster&display=swap" rel="stylesheet">
         <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+TC:wght@500&display=swap" rel="stylesheet">
+
+        <!-- js -->
+        <script src="https://www.gstatic.com/charts/loader.js"></script>
+        <script type="text/javascript" src="./js/jquery-3.0.0.js"></script>
+        <script src="./js/addheader.js"></script>
+        <script src="./js/index.js"></script>
```

Comment:
You can actually put these scripts at the bottom of the body tag, since loading these scripts would block the page rendering, better to show something to user before blocking.